tp1_directgeom/main.asciidoc

```diff
@@ -136,9 +136,9 @@ pkg  = '/home/student/models/'
 urdf = pkg + '/ur_description/urdf/ur5_gripper.urdf'
 robot = RobotWrapper(urdf,[pkg,])
 ----
-The code of the RobotWrapper class is in /path/to/pinocchio//src/python/romeo_wrapper.py . Do not hesitate to have a look at it and to take inspiration from the implementation of the class functions.
+The code of the RobotWrapper class is in /opt/openrobots/lib/python2.7/site-packages/pinocchio/robot_wrapper.py . Do not hesitate to have a look at it and to take inspiration from the implementation of the class functions.
```

Comment:
We should not over-specified the tuto toward robotpkg installation.
I suggest: the code of the RobotWrapper class is in the Pinocchio source package (in pinocchio/bindings/python/script/robot_wrapper.py) and is installed when compiling the package in lib/python2.7/site-packages/pinocchio/robot_wrapper.py (when installing using robotpkg or inside the virtual-box, the exact path is /opt/openrobots/lib/python2.7/site-packages/pinocchio/robot_wrapper.py ).

Reply:
if the $PYTHONPATH is correctly set, references to robotpkg should not be needed.